pallets/ocex/src/lib.rs

```diff
@@ -1233,6 +1232,70 @@ pub mod pallet {
 				false
 			}
 		}
+
+		pub fn get_ob_recover_state() -> Result<
```

Comment:
Please add some documentation for the public api.

Reply:
done